src/app/qgsmeasuredialog.cpp

```diff
@@ -56,9 +57,15 @@ QgsMeasureDialog::QgsMeasureDialog( QgsMeasureTool *tool, Qt::WindowFlags f )
 
   if ( !mMeasureArea )
   {
+    QAction *copyAction = new QAction( tr( "Copy &All" ), this );
```

Comment:
Why not calling it "copy" since there are not alternatives to copying all. Makes the button narrower and fits a bit better alongside the other 1-word buttons.

Reply:
Good point. Will do!